scaleway/resource_rdb_instance_beta.go

```diff
@@ -234,13 +236,26 @@ func resourceScalewayRdbInstanceBetaUpdate(d *schema.ResourceData, m interface{}
 	if err != nil {
 		return err
 	}
-
+	var upgradeInstanceRequests []rdb.UpgradeInstanceRequest
 	if d.HasChange("node_type") {
-		_, err = rdbAPI.UpgradeInstance(&rdb.UpgradeInstanceRequest{
-			Region:     region,
-			InstanceID: ID,
-			NodeType:   scw.StringPtr(d.Get("node_type").(string)),
-		})
+		upgradeInstanceRequests = append(upgradeInstanceRequests,
+			rdb.UpgradeInstanceRequest{
+				Region:     region,
+				InstanceID: ID,
+				NodeType:   scw.StringPtr(d.Get("node_type").(string)),
+			})
+	}
+
+	if d.HasChange("is_ha_cluster") {
+		upgradeInstanceRequests = append(upgradeInstanceRequests,
+			rdb.UpgradeInstanceRequest{
+				Region:     region,
+				InstanceID: ID,
+				EnableHa:   scw.BoolPtr(d.Get("is_ha_cluster").(bool)),
+			})
+	}
+	for _, request := range upgradeInstanceRequests {
```

Comment:
Can't this be done in a single upgrade request ? 
iirc, the second call will fail since the db will be in initializing state